Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst

```diff
@@ -0,0 +1,116 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_mutual_authentication:
+
+*********************
+Mutual Authentication
+*********************
+
+.. include:: ../../../beta.rst
+
+Mutual TLS Requirements
```

Comment:
```suggestion
Mutual Authentication and mTLS background
```

tbh I think this whole file should probably be trimmed a lot, and basically say something along the lines of:
- the Service Mesh feature here is mTLS-based mutual authentication
- We call it this because we split the handshake from the record protocol
- For Cilium to meet most of the usual requirements for service-to-service authentication and encryption, users need to enable encryption, see <this section of the docs>
- We should refer more discussion about "Why SPIFFE" etc to the CFP in the `design-cfps` repo.